docs/aws/karpenter.cloudformation.yaml

```diff
@@ -0,0 +1,88 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: Resources used by https://github.com/awslabs/karpenter
+Parameters:
+  OpenIDConnectIdentityProvider:
+    Type: String
+    Description: "Example oidc.eks.us-west-2.amazonaws.com/id/1234567890. "
```

Comment:
is the `.` at end maybe going to be confusing?

Reply:
+1